python/lsst/ts/externalscripts/make_love_stress_tests.py

```diff
@@ -0,0 +1,341 @@
+        self.config = None
+
+        # list to store clients connections,
+        # each one an instance of ManagerClient
+        self.clients = []
+
+        # dict to store remote connections,
+        # with each item in the form of
+        # `CSC_name[:index]`: `lsst.ts.salobj.Remote`
+        self.remotes = {}
+
+        # commands timeout
+        self.cmd_timeout = 10
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_standardscripts/StressLOVE.yaml
+            title: StressLOVE v1
+            description: Configuration for StressLOVE
+            type: object
+            properties:
+              location:
+                description: Domain of the running LOVE instance to stress
+                type: string
+              number_of_clients:
+                description: The number of clients to create
+                type: number
+              number_of_messages:
+                description: The number of messages to store before calculating the mean latency
+                type: number
+              data:
+                description: List of CSC_name[:index]
+                type: array
+                minItems: 1
+                items:
+                    type: string
+            required: [location, number_of_clients, number_of_messages, data]
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    def set_metadata(self, metadata):
+        """Compute estimated duration.
+
+        Parameters
+        ----------
+        metadata : SAPY_Script.Script_logevent_metadataC
+        """
+        # a crude estimate;
+        metadata.duration = len(self.clients) * 5
+
+    async def configure(self, config):
+        """Configure the script.
+
+        Specify the Stress test configurations:
+        - LOVE host location
+        - Number of clients
+        - Number of messages to store
+        - CSCs
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Configuration with several attributes, defined in `get_schema`
+
+        Notes
+        -----
+        Saves the results on several attributes:
+
+        * location : a string, with the location of the LOVE instance to stress
+        * config : a dict, with the number_of_clients and number
+            of_messages configurations
+        * remotes : a dict, with each item as
+            CSC_name[:index]: `lsst.ts.salobj.Remote`
+
+        Constructing a `salobj.Remote` is slow (DM-17904), so configuration
+        may take a 10s or 100s of seconds per CSC.
+        """
+        self.log.info("Configure started")
+
+        # set configurations
+        self.location = config.location
+        self.config = {
+            "number_of_clients": config.number_of_clients,
+            "number_of_messages": config.number_of_messages,
+        }
```

Comment:
I strongly recommend simplifying the lines (all of # set configurations) to:
```
self.config = config
```
Then you can access location as `self.config.location`, number_of_clients as `self.config.number_of_clients`, etc. It avoids repetition and it eliminates the need to use dict style access.